cmd/bosun/sched/template.go

```diff
@@ -295,26 +298,28 @@ func (c *Context) graph(v interface{}, filter bool) (interface{}, error) {
 			Filename:    name,
 			ContentType: "image/png",
 		})
-		return template.HTML(fmt.Sprintf(`<img alt="%s" src="cid:%s" />`,
+		return template.HTML(fmt.Sprintf(`<img alt="%s" src="cid:%s" />%s`,
 			template.HTMLEscapeString(fmt.Sprint(v)),
 			name,
+			footerHTML,
 		)), nil
 	}
-	if err := c.schedule.ExprSVG(nil, &buf, width, height, res, title, time.Now().UTC()); err != nil {
+	if err := c.schedule.ExprSVG(nil, &buf, width, height, unit, res); err != nil {
 		return nil, err
 	}
+	buf.WriteString(footerHTML)
 	return template.HTML(buf.String()), nil
 }
 
 // Graph returns an SVG for the given result (or expression, for which it gets the result)
 // with same tags as the context's tags.
-func (c *Context) Graph(v interface{}) (interface{}, error) {
-	return c.graph(v, true)
+func (c *Context) Graph(v interface{}, unit string) (interface{}, error) {
```

Comment:
Having `unit` as the last argument means you can't do some things you could before with the go templates (see the [pipelines section](http://golang.org/pkg/text/template/#hdr-Pipelines) in the godocs). For example, here's a line copied from our prod.conf:

`{{printf "q(\"avg:rate{counter,,1}:os.cpu{host=%s}\", \"1d\", \"\")" (.Group.host) | .Graph}}`

This won't work anymore because you can only pass the last argument of a function via `|`.

One partial fix is to make `Graph` and `GraphAll` take a `...interface{}`. Then we could check the types and counts of things and handle them correctly. This would not break any existing syntax, however you still couldn't pass a y_label string to the above call.

One other possibility is to swap the order of the arguments.

We could also combine both solutions: swap and allow varargs. If the number of arguments is 1, then it's an expression. If it's 2, the first is a unit, the second is the expression.